source/fundamentals/authentication.txt

```diff
@@ -153,12 +153,24 @@ MONGODB-AWS
    The ``MONGODB-AWS`` authentication mechanism is available only for
    MongoDB deployments on MongoDB Atlas.
 
-The ``MONGODB-AWS`` authentication mechanism uses your Amazon Web Services
-Identity and Access Management (AWS IAM) credentials to authenticate your
-user. You can either specify your credentials explicitly
-or instruct the driver to retrieve them automatically from an external source.
+The ``MONGODB-AWS`` authentication mechanism uses Amazon Web Services
+Identity and Access Management (AWS IAM) credentials to authenticate a user to MongoDB.
 
-The following sections contain code examples that use the following placeholders:
+To use AWS authentication, first perform the following steps:
+
+- Add the ``MongoDB.Driver.Authentication.AWS`` NuGet package to your project.
+- Add the following line of code to your application's bootstrap code to register the
+  AWS authentication provider:
+
+  .. code-block:: csharp
+
+     MongoAuthentication.RegisterAwsAuthenticationProvider();
```

Comment:
Replace the code with the following one:
`MongoClientSettings.Extensions.AddAWSAuthentication();`

Reply:
That's really weird. I don't know where that line came from.